SPECS/icingaweb2-module-director.spec

```diff
@@ -64,6 +64,13 @@ director_sec=$(awk '/\[director\]/{flag=1;next}/\[/{flag=0}flag' /etc/icingaweb2
 mysql_director_name=$(grep 'dbname' <<< "$director_sec" | cut -d'=' -f2 | tr -d "\"'" | xargs)
 mysql_director_user=$(grep 'username' <<< "$director_sec" | cut -d'=' -f2 | tr -d "\"'" | xargs)
 mysql_director_psw=$(grep 'password' <<< "$director_sec" | cut -d'=' -f2 | tr -d "\"'" | xargs)
+
+if /usr/bin/icingacli director migration pending ; then
+  /usr/bin/icingacli director migration run
```

Comment:
`/usr/bin` is not used thoughout the specfile for `icingacli`. I suggest removing it for consistency. Otherwise it looks fine, looking at `icingacli director migration pending --help`.